### PR TITLE
WarpX 23.02

### PR DESCRIPTION
**Pull request by ax3l**
Update `warpx` & `py-warpx` to the latest release.

### CODE REVIEW

**Review by tldahlgren — COMMENTED** [review]
Confirmed both version sha256.

**Review by tldahlgren — APPROVED** [review]

**Review by adamjstewart — APPROVED** [review]

**Comment by adamjstewart** [comment]
Do you want to wait for the other maintainers to review or should we just merge it since it's a minor patch release?

**Review by dpgrote — APPROVED** [review]
I can approve. Thanks!

**Pull request merged**
